libs/Microsoft.MixedReality.WebRTC/Interop/InteropUtils.cs

```diff
@@ -39,11 +39,7 @@ sealed class MonoPInvokeCallbackAttribute : Attribute
 
     internal static class Utils
     {
-#if MR_SHARING_WIN
-        internal const string dllPath = "Microsoft.MixedReality.WebRTC.Native.dll";
-#elif MR_SHARING_ANDROID
-        internal const string dllPath = "Microsoft.MixedReality.WebRTC.Native.so";
-#endif
+        internal const string dllPath = "Microsoft.MixedReality.WebRTC.Native";
```

Comment:
Unfortunately this needs to be reverted, this makes DLL loading fail on Windows because the filename has dots in its name. See https://github.com/dotnet/runtime/issues/7223. I think we talked about it offline, or you mentioned it to me, but I just hit it and couldn't run `TestAppUWP.exe` anymore to test the connection.

Reply:
This change made it possible to include a single copy of the DLL in Unity and it would load for multiple platforms (Windows Desktop and Android), but if that doesn't work in UWP then it has to be reverted. The consequence is that Unity projects will need to have multiple copies of this DLL present - one for the .dll extension (Windows/UWP) and another for the .so extension (Android) - each configured to load for that platform only.

Reply:
Ah, right. But since they deploy to difference platforms, that has no impact on the final app size on each device, correct? So I guess it's just a matter of having 2 files instead of 1 in your Unity project. Am I missing something here?

Reply:
That sounds right to me.

Reply:
Fixed by renaming `Microsoft.MixedReality.WebRTC.Native.dll` into `mrwebrtc.dll` / `libmrwebrtc.so`, which both work with P/Invoke as `[DllImport("mrwebrtc")]`.